NGit/NGit.Api.Errors/PatchApplyException.cs

```diff
@@ -65,5 +65,9 @@ public class PatchApplyException : GitAPIException
 		public PatchApplyException(string message) : base(message)
 		{
 		}
-	}
+
+		public PatchApplyException()
+        {
+        }
```

Comment:
You've got some weird formatting going on here, but then so has the rest of the file.